app/src/main/java/protect/card_locker/AboutActivity.java

```diff
@@ -95,83 +96,61 @@ protected void onCreate(Bundle savedInstanceState)
         {
             Log.w(TAG, "Package name not found", e);
         }
-
         setTitle(String.format(getString(R.string.about_title_fmt), appName));
 
-
-
-//        TextView aboutTextView = findViewById(R.id.aboutText);
-//        aboutTextView.setText(HtmlCompat.fromHtml(String.format(getString(R.string.debug_version_fmt), version) +
-//                "<br/><br/>" +
-//                String.format(getString(R.string.app_revision_fmt),
-//                        "<a href=\"" + getString(R.string.app_revision_url) + "\">" +
-//                                "GitHub" +
-//                                "</a>") +
-//                "<br/><br/>" +
-//                String.format(getString(R.string.app_copyright_fmt), year) +
```

Comment:
This line puts the correct year in app_copyright_fmt, you want to use something like this for the new version history subtext.